docs/examples/cart-service/part-1-build-and-deploy.md

```diff
@@ -134,15 +134,16 @@ export class Cart {
 
   // Helper function that creates a copy of the current cart items and adds the new one
   public static newItems(items: Array<CartItem>, sku: string, quantity: number): Array<CartItem> {
-    return items.map(item => {
-      if (item.sku == sku) {
-        return {
-          ...item,
-          quantity: item.quantity + quantity
-        }
-      }
-      return {...item}
-    })
+    let current_item = items.find(i => i.sku === sku)
+    if (!current_item) {
```

Comment:
would it make it simpler to read if we get rid of the `!` and flip the code?

Reply:
Done 32f4016da82fd6c042ea693f7bfe7e71455b479e